SimpleKeychain/A0SimpleKeychain.m

```diff
@@ -356,12 +356,12 @@ - (NSDictionary *)queryFetchOneByKey:(NSString *)key message:(NSString *)message
                                       (__bridge id)kSecAttrAccount: key,
                                       }];
 #if TARGET_OS_IPHONE
-    if (self.useAccessControl && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
-        if (message) {
+    if (self.useAccessControl) {
+        if (message && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
             query[(__bridge id)kSecUseOperationPrompt] = message;
+        } else {
+            NSLog(@"Error trying to access to non available kSecUseOperationPrompt in iOS7");
```

Comment:
Yeah, the thing is that is not a "real error" more like a warning, let's keep it but changing it a bit
```objc
#ifdef DEBUG
else {
    NSLog(@"Warning: Ignoring use of kSecUseOperationPrompt that is not available in iOS 7");
}
#endif
```

On a side note, I'll research if I can mark some methods/properties as disabled for specific iOS version so the compiler will complain instead.